rgw/v2/tests/s3_swift/test_swift_basic_ops.py

```diff
@@ -418,6 +443,54 @@ def test_exec(config, ssh_con):
                                 f"Get object with different tenant of with same user failed as expected: {e}"
                             )
 
+                    if config.test_ops.get(
+                        "upload_another_large_object_with_same_name_with_diff_tenants",
+                        False,
+                    ):
+                        log.info(
+                            f"Upload large object {swift_object_name} again with container owner"
+                        )
+                        large_object_name = fill_container(
+                            rgw,
+                            container_name,
+                            user_name,
+                            oc,
+                            cc,
+                            size,
+                            multipart=True,
+                            split_size=config.split_size,
+                            swift_object_name=swift_object_name,
+                        )
+                        if swift_object_name != large_object_name:
+                            raise TestExecError(
+                                f"Try Upload large object:{swift_object_name} twice failed, but uploaded {large_object_name}"
+                            )
+
+                        if config.test_ops.get("new_tenant", False):
+                            log.info(
+                                f"Upload large object:{large_object_name},  with different tenant user"
+                            )
+                            # Verify same user in different tenant not having permission for container can upload objects
```

Comment:
can we make the statement little clear..As I understood user from different tenant cannot upload objects..

Reply:
updated